specification/appcomplianceautomation/AppComplianceAutomation.Management/resources/SnapshotResource.tsp

```diff
@@ -0,0 +1,79 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "../models/models.tsp";
+import "./ReportResource.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using OpenAPI;
+
+namespace Microsoft.AppComplianceAutomation;
+/**
+ * A class represent a AppComplianceAutomation snapshot resource.
+ */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "For backward compatibility"
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property" "For backward compatibility"
+@parentResource(ReportResource)
+@includeInapplicableMetadataInPayload(false)
+@Azure.ResourceManager.Private.armResourceInternal(SnapshotProperties)
+model SnapshotResource extends Azure.ResourceManager.Foundations.ProxyResource {
+  /**
+   * Snapshot Name.
+   */
+  @path
+  @key("snapshotName")
+  @segment("snapshots")
+  @visibility("read")
+  @pattern("^[a-zA-Z0-9-_]{1,64}$")
+  name: string;
+
+  /**
+   * Snapshot's property.
+   */
+  @extension("x-ms-client-flatten", true)
+  @extension("x-ms-client-name", "properties")
+  properties?: SnapshotProperties;
+}
+
+@armResourceOperations
+@tag("AppComplianceAutomation")
+interface SnapshotResources {
```

Comment:
Same with other interface, better to remove suffix `Resources`

Reply:
Removed the suffix Resources.